core/api/src/test/java/com/microsoft/gctoolkit/test/time/DateTimeStampTest.java

```diff
@@ -353,5 +357,20 @@ void timeSpanInMinutes() {
         diff = a.timeSpanInMinutes(b);
         assertEquals((.18 - .19)/60d, diff, .001);
     }
+    @Test
+    void testCompare(){
```

Comment:
Prefer tests that have one assert so perhaps test less than, equals to and greater than?